dccrg.hpp

```diff
@@ -9734,7 +9747,7 @@ template <
 					this->neighborhood_of,
 					2 * this->max_ref_lvl_diff,
 					true
-				);
+				);	// todo this should probably be using cached values
```

Comment:
But so will this also save/gain us some performance? Should this be tested then?

Reply:
Possibly. Needs a benchmark on an AMR run, iirc refines/unrefines are instrumented now?